tests/test_crud_data.py

```diff
@@ -0,0 +1,373 @@
+
+        data = result_to_dict(crud.get(3))
+        self.assertNotEqual(data, None)
+        self.assertDictEqual(data, {'id': 3, 'name': 'Sam', 'email': 'sam@def.com', 'active': True})
+        self.assertEqual(data['id'], 3)
+        self.assertEqual(data['name'], 'Sam')
+        self.assertEqual(data['email'], 'sam@def.com')
+        self.assertEqual(data['active'], True)
+
+        data = result_to_dict(crud.get(4))
+        self.assertNotEqual(data, None)
+        self.assertDictEqual(data, {'id': 4, 'name': 'Jinny', 'email': 'jinny@def.com', 'active': True})
+        self.assertEqual(data['id'], 4)
+        self.assertEqual(data['name'], 'Jinny')
+        self.assertEqual(data['email'], 'jinny@def.com')
+        self.assertEqual(data['active'], True)
+
+        data = result_to_dict(crud.get(5))
+        self.assertNotEqual(data, None)
+        self.assertDictEqual(data, {'id': 5, 'name': 'Rosa', 'email': 'rosa@def.com', 'active': True})
+        self.assertEqual(data['id'], 5)
+        self.assertEqual(data['name'], 'Rosa')
+        self.assertEqual(data['email'], 'rosa@def.com')
+        self.assertEqual(data['active'], True)
+
+        crud.update(1, {'active': False})
+
+        data = result_to_dict(crud.get(1))
+        self.assertNotEqual(data, None)
+        self.assertDictEqual(data, {'id': 1, 'name': 'Jon', 'email': 'jon@def.com', 'active': False})
+        self.assertEqual(data['id'], 1)
+        self.assertEqual(data['name'], 'Jon')
+        self.assertEqual(data['email'], 'jon@def.com')
+        self.assertEqual(data['active'], False)
+
+        crud.update(5, {'name': 'Rosita', 'email': 'rosita@def.com'})
+
+        data = result_to_dict(crud.get(5))
+        self.assertNotEqual(data, None)
+        self.assertDictEqual(data, {'id': 5, 'name': 'Rosita', 'email': 'rosita@def.com', 'active': True})
+        self.assertEqual(data['id'], 5)
+        self.assertEqual(data['name'], 'Rosita')
+        self.assertEqual(data['email'], 'rosita@def.com')
+        self.assertEqual(data['active'], True)
+
+        crud.delete(2)
+        with self.assertRaises(StatusCodeException):
+            crud.get(2)
+
+        crud.delete(3)
+        with self.assertRaises(StatusCodeException):
+            crud.get(3)
+
+        crud.delete(4)
+        with self.assertRaises(StatusCodeException):
+            crud.get(4)
+
+    def test_crud_soft_delete(self):
+        crud = CRUDSoftDeleteInit()
+
+        crud.create({'name': 'Jon', 'email': 'jon@def.com', 'active': True})
+        crud.create({'name': 'Ron', 'email': 'ron@def.com', 'active': True})
+        crud.create({'name': 'Sam', 'email': 'sam@def.com', 'active': True})
+        crud.create({'name': 'Jinny', 'email': 'jinny@def.com', 'active': True})
+        crud.create({'name': 'Rosa', 'email': 'rosa@def.com', 'active': True})
+
+        data = result_to_dict(crud.get(1))
+        self.assertNotEqual(data, None)
+        self.assertEqual(data['id'], 1)

```

Comment:
its better to compate to a dict as in the above. will cover sases of new properties